arbnode/seq_coordinator.go

```diff
@@ -247,8 +259,27 @@ func (c *SeqCoordinator) chosenOneUpdate(ctx context.Context, msgCountExpected,
 	if err != nil {
 		return err
 	}
+	safeLockoutElapse := lockoutUntil.Add(-c.config.LockoutSpare)
+	if time.Now().After(safeLockoutElapse) {
+		return fmt.Errorf("safe lockout elapsed %v in the past", time.Since(safeLockoutElapse))
+	}
+	if c.delayedSequencer != nil && !hadLockout {
```

Comment:
this is too deep down.
ForceSequencerDelay will eventually call chosenOneUpdate again, which seems unhealthy. It fits better in the update loop.

Reply:
Refactored :)